### PR TITLE
Document the existing package manager for MacOS homebrew

### PR DESCRIPTION
**Pull request by norman-abramovitz**
Resolves #

### CODE REVIEW

File: website/content/docs/install.mdx [hunk 1]
```diff
@@ -20,8 +20,16 @@ There are several options to install OpenBao:
 
 ## Package manager
 
+### Homebrew - MacOS
+
+```shell-session
+$ brew info openbao
+$ brew install openbao
+```
+
```
Comment (cipherboy) [inline website/content/docs/install.mdx:29]: Not to nitpick too terribly much, but do you mind adding a heading so it is clear the below is for Linux? Thanks!

```suggestion

### Linux

```

**Comment by cipherboy** [comment]
Thanks!

**Review by cipherboy — APPROVED** [review]

**Comment by norman-abramovitz** [comment]
I just saw the request to add a header for Linux, and I am glad I double-checked to make sure the PR was still open.   

The Apple Branding convention is some version of macOS. 
https://en.wikipedia.org/wiki/MacOS#:~:text=Apple%20shortened%20the%20name%20to,iOS%2C%20watchOS%2C%20and%20tvOS.

I am not sure calling  "macOS" "Linux" is appropriate.   I would do whatever the OpenBao maintainers want.  WIth the PR being merged, there is nothing to do.   No response is need if my assumption is correct.

**Comment by cipherboy** [comment]
@norman-abramovitz Sorry, the header was below your addition of MacOS instructions -- the text under it was talking about Linux distribution packaging, so the MacOS header would be confusing. 

Let me know if you think something still needs to be changed:

image

**Comment by norman-abramovitz** [comment]
I am good with it.  Another reasonable title I can think of is  "Other Package Managers" to be inclusive for Windows.  ROTFL.

**Pull request merged**